ArduCopter/mode_autotune.cpp

```diff
@@ -38,30 +38,20 @@ void AutoTune::run()
     // apply SIMPLE mode transform to pilot inputs
     copter.update_simple_mode();
 
-    // reset target lean angles and heading while landed
+    // disarm when the landing detector says we've landed and spool state is ground idle
+    if (copter.ap.land_complete && motors->get_spool_state() == AP_Motors::SpoolState::GROUND_IDLE) {
+        copter.arming.disarm(AP_Arming::Method::LANDED);
+    }
+
+    // if not armed set throttle to zero and exit immediately
     if (copter.ap.land_complete) {
-        // we are landed, shut down
-        float target_climb_rate = get_pilot_desired_climb_rate_cms();
-
-        // set motors to spin-when-armed if throttle below deadzone, otherwise full range (but motors will only spin at min throttle)
-        if (target_climb_rate < 0.0f) {
-            copter.motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::GROUND_IDLE);
```

Comment:
@rmackay9 I suppose this would allow users to takeoff.  however this is far from the current code in althold and loiter used for spool state management and takeoff.

 IMHO, I don't think it is a good/safe practice to allow takeoff in autotune. @lthall what do you think?

Reply:
The first twitch of autotune can be very large and aggressive. If we let people take-off in autotune and they hover at 1m for a second that first twitch could easily cause a crash.

Given we switch to Autotune from Loiter or Alt_Hold to determine what behaviour we use I don't think we should be starting in Autotune.

Reply:
In case it matters, I think this is an existing issue